ultimatejobweb/migrations/0001_initial.py

```diff
@@ -27,7 +27,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('profession', models.CharField(max_length=128)),
-                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE,to=settings.AUTH_USER_MODEL)),
```

Comment:
why did you change this file? It is auto-generated and should be changed

Reply:
I just changed all files names as you requested.
Fixed the auto-generated Python file, too, and now it has no problems with flake8. (I changed it at first because flake8 showed there was too many characters in line 30, but now it is working well.
Is the PR approved? @simzacks